_blogposts/2020-09-23-release-8-3.md.raw

```diff
@@ -0,0 +1,119 @@
+---
+author: hongbo
+date: "2020-09-23" 
+previewImg:
+category: compiler
+title: What's new in ReScript 8.3 (Part 1)
+description: |
+---
+
+
+# What's new in ReScript 8.3 (Part 1)
+
+ReScript is a soundly typed language with an optimizing compiler focused on JS platform. 
+It's focused on type safety, performance and JS interop. It used to be called BuckleScript.
+
+ReScript@8.3 is available, people can try it via
+
+```
+npm i bs-platform@8.3.0
+```
+
+The changes are listed [here](https://github.com/rescript-lang/rescript-compiler/blob/master/Changes.md#83), this is a large release,  and we will go through some highlighted changes.
+
+
+
+## Lightweight FFI attributes without `bs.` prefix
+
+In this release, we make it optional to have `bs.` prefix it or not, this will make the FFI less verbose,
+
+For example, the old externals for `readFileAsUtf8Sync` used to be written like this
+
+
+```ocaml
+external readFileAsUtf8Sync : string -> (_[@bs.as "utf8"]) -> string = "readFileSync" [@@bs.val] [@@bs.module "fs"]
+```
+
+Could be simplified as 
+```ocaml
+external readFileAsUtf8Sync : string -> (_[@as "utf8"]) -> string = "readFileSync" 
+[@@val] [@@module "fs"]
+```
+
+Note almost all previous attributes with `bs.xx` can be simplified as `xx`, except such two that 
+we don't provide abbreviations:
+
```

Comment:
Note that almost all previous attributes with `bs.xx` can be simplified as `xx` with the exception of the following two that don't have abbreviations: